src/licensedcode/data/rules/elastic-license-v2_or_agpl-3.0_or_mongodb-sspl-1.0_1.RULE

```diff
@@ -0,0 +1,17 @@
+---
+license_expression: elastic-license-v2 OR agpl-3.0 OR mongodb-sspl-1.0
+is_license_notice: yes
+relevance: 100
+notes: https://github.com/elastic/elasticsearch/commit/a59c182f9f7e9d1bf3d6eecbc0e44f24ff91d053
+ignorable_copyrights:
+    - Copyright Elasticsearch B.V.
+ignorable_holders:
+    - Elasticsearch B.V.
+---
+
+* Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "{{Elastic License}}
+ * 2.0", the "{{GNU Affero General Public License v3.0 only}}", and the "{{Server Side 
```

Comment:
Include the 2.0 in the required phrase: "{{Elastic License 2.0}}"
Also add the required phrase to all the instances of license names over this rules and other rules you've added to this PR.